public/app/features/explore/Explore.tsx

```diff
@@ -220,7 +220,7 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
         modifiedQueries = [...this.modifiedQueries];
       } else if (datasource.importQueries) {
         // Datasource-specific importers
-        modifiedQueries = await datasource.importQueries(this.modifiedQueries, origin.meta);
+        modifiedQueries = await datasource.importQueries(this.modifiedQueries, datasource.meta);
```

Comment:
This should stay `origin` no? The point of the importers is to see if target DS has an importer that understands queries from an origin DS.